src/test/resources/docker/Dockerfile.sshagent

```diff
@@ -5,6 +5,9 @@ RUN apk update && \
     apk upgrade && \
     apk add openssh su-exec && \
     rm /var/cache/apk/* && \
+    if [ "$testuid" -gt 0 ]; then if egrep "^[^:]+:x:$testuid:" /etc/passwd; then deluser "$(egrep "^[^:]+:x:$testuid:" /etc/passwd | cut -d: -f1)"; fi; fi && \
+    if [ "$testgid" -gt 0 ]; then if egrep "^[^:]+:x:[^:]+:$testgid:" /etc/passwd; then deluser "$(egrep "^[^:]+:x:[^:]+:$testgid:" /etc/passwd | cut -d: -f1)"; fi; fi && \
+    if [ "$testgid" -gt 0 ]; then if egrep "^[^:]+:x:$testgid:" /etc/group; then delgroup "$(egrep "^[^:]+:x:$testgid:" /etc/group | cut -d: -f1)"; fi; fi && \
```

Comment:
The issue with the latest Ubuntu 22.04 based Github runners turned out to be that the user inside the Azure VM had a GID of `123`, which conflicted with the built-in `ntp` group inside the Alpine container.
This caused the `addgroup` command below to fail, since it couldn't add a `testuser` group with the same GID.
The gross hack above is to check for any UID/GID conflicts inside the Alpine container and remove the conflicting users and/or groups.